app/assets/javascripts/discourse-markdown-it/src/setup.js

```diff
@@ -0,0 +1,360 @@
+import deprecated from "discourse-common/lib/deprecated";
```

Comment:
This was previously `setupIt` in `pretty-text/addon/engines/discourse-markdown-it.js`